protobuf/CMakeLists.txt

```diff
@@ -63,11 +63,10 @@ if(${TRITON_COMMON_ENABLE_PROTOBUF})
       $<BUILD_INTERFACE:${CMAKE_CURRENT_BINARY_DIR}>
   )
 
-  target_compile_options(
-    proto-library PRIVATE
-    $<$<OR:$<CXX_COMPILER_ID:Clang>,$<CXX_COMPILER_ID:AppleClang>,$<CXX_COMPILER_ID:GNU>>:
-      -Wall -Wextra -Wno-unused-parameter -Werror>
```

Comment:
This options are responsible for:
[`-Wall`](https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wall) - enable all warnings 
[`-Wextra`](https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-W) - enable extra warnings 
[`-Wno-unused-parameter`](https://gcc.gnu.org/onlinedocs/gcc-3.2/gcc/Option-Summary.html#Option%20Summary) - can't find definition
[`-Werror`](https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Werror) - make all warnings as error